ws-language/templates/recommendations.md

```diff
@@ -1,7 +1,32 @@
-# Recommendation template
+# Proposed template for Inclusive Naming Initiative term guidance
+Julian Cable, May 2021
 
+Each term is assigned to one of the following categories:
+* Preferred: Use this term.
+* Use with caution: Use the term only in a specified context; otherwise use a stated alternative.
+* Do not use: Never use the term; instead use a stated alternative.
+The categories might be denoted with words, or an icon, or both.
 
-| title | title | title | 
-| --- | --- | --- |
-| cell | cell | cell |
-| cell | cell | cell |
+Suggested template for term guidance:
+
+| Term | Status | Part of speech | Usage note | 
+| --- | --- | --- | --- |
+| Term A | Preferred | noun | Use to refer to ... | 
+| Term B | Preferred | verb | Write as shown: not "[disallowed-term]". | 
+| Term C | Preferred | adjective | Write as shown: two words, lowercase. | 
+| Term D | Preferred | adverb | Write as shown: one word, unhyphenated. | 
+| Term E | Preferred | noun | Write as shown, with initial uppercase letter. | 
+| Term F | Use with caution | noun | Use only to refer to ... Otherwise, use [preferred-term]. [Justification] | 
+| Term G | Use with caution | verb | Do not use to describe ... [Justification] | 
+| Term H | Use with caution | adjective, noun | Do not use in a context where ... [Justification] |
+| Term I | Do not use | noun | Do not use. Use "[preferred-term]". [Justification] |
+| Term J | Do not use | noun | Do not use. Use another term, such as "[option-1]" or "[option-2]". [Justification] |
+
+Example entries:
+| Term | Status | Part of speech | Usage note | 
+| --- | --- | --- | --- |
+| Indigenous | Preferred | adjective | Write as shown, with initial uppercase I, to refer to people who are native to a region or country. |
+| master | Use with caution | adjective, noun | Do not use when it is paired with "slave", explicitly or implicitly; such use evokes the dehumanizing practice of slavery. Use alternative terms such as "primary" and "secondary", or "[alternative-term1]" and "[alternative-term2]" , or "[alternative-term3]" and "[alternative-term4]". However, use of "master" is acceptable in other contexts, such as to refer to mastery of a skill. |
+| segregation | Do not use | noun | Do not use. Use "separation" or "segmentation". This term refers to a process that violates human rights by limiting access to privileges and rights such as education and voting. |
```

Comment:
In this example verbiage, which is good to have BTW, I would opt not to include some of the candidate replacement terms and instead use the proxy text `alternative-term-1` and `alternative-term-2`. I think its far too likely and easy that others stumbling here will read this less as an example of how to assert guidance and more as actual guidance.